build.gradle

```diff
@@ -1,16 +1,39 @@
 plugins {
   id 'java'
+  id 'net.researchgate.release' version '2.7.0'
+  id 'com.jfrog.bintray' version '1.8.0'
 }
 
 allprojects {
+  apply plugin: 'maven-publish'
+  apply plugin: 'com.jfrog.bintray'
+
   repositories {
     jcenter()
   }
 
+  release {
+    tagTemplate = 'v${version}'
+  }
+
   sourceCompatibility = 1.8
   targetCompatibility = 1.8
 }
 
 task wrapper(type: Wrapper) {
   gradleVersion = '4.7'
 }
+
+task verifyBintrayConfig {
+  doLast {
+    println "bintray_user: $bintray_user"
+    println "bintray_apikey: $bintray_apikey"
+    println "bintray_dryrun: ${bintray_dryrun}"
+  }
+}
+
+task imWithStupid(type:Exec) {
+  commandLine './gradlew', '-info', 'bintrayUpload'
+}
+
+afterReleaseBuild.dependsOn imWithStupid
```

Comment:
There's something stupid going on with the `gradle-bintray-plugin` if I depend on `bintrayUpload` gradle says it can't find the tasks dependencies and freaks out. So what's happening here is that I'm shelling out after the tag is made and the binaries are built to upload them to bintray. If we wait until after the release process is done we'll be on the snapshot, which is not what we want.

Reply:
@pdex - that's probably because the tasks don't exist a the root level - see my comment above about moving these changes into the xio build.gradle